aws/alarms/inputs.tf

```diff
@@ -63,3 +63,23 @@ variable "threshold_lb_response_time" {
   description = "Load balancer response time, in seconds, above which an alarm is triggered (10 minute period)"
   type        = string
 }
+
+variable "sns_topic_alert_critical_arn" {
+  type = string
```

Comment:
nit: could you add a `description` to each of these new variables that describes what it is.  Just something simple like:
```hcl
description = "SNS topic ARN that critical alerts are sent to"
```